package.json

```diff
@@ -8,10 +8,11 @@
     "babel-server": "babel server/ -d dist/server/ && npm run babel-app",
     "build": "npm run clean && mkdir dist/ && npm run babel-server",
     "clean": "rm -rf dist/",
-    "test": "mocha --require babel-register server/tests/*.test.js",
+    "test": "mocha --require babel-core/register ./server/tests/*.test.js  --exit",
```

Comment:
whats happening here?

Reply:
noted. Thanks

Reply:
What are you noting please? I'd like a response to the question. To be specific why do we need the `--exit` flag and the `babel-core` change

Reply:
the --exit flag is to end/exit the test after running. the babel-core change was a mistake. from the initial command on the script "mocha --require babel-core/register", the command is to pick either pick babel-core or babel-register depending on which one is available. so I checked my package.json and picked babel-register because it was in the package.json.